lib/cli-engine.js

```diff
@@ -95,7 +95,8 @@ var defaultOptions = {
         // it will always be used
         cacheLocation: "",
         cacheFile: ".eslintcache",
-        fix: false
+        fix: false,
+        inlineConfig: true
```

Comment:
`--no-inline-config` is `false`, meaning `inlineConfig` is `true`. I used `--no-inline-config` in the command line docs as it's a nicer, more straightforward API. But internally the behaviour is `inlineConfig` is always true/on unless explicitly marked as `false` by either the CLI or a `CLIEngine` config.